src/js/components/Select/stories/theme.js

```diff
@@ -12,7 +10,7 @@ export const theme = {
       selected: 'neutral-3',
       border: '#e0e0e0',
       focus: '#2196F3',
-      gray: rgba(0, 0, 0, 0.54),
+      gray: 'rgba(0, 0, 0, 0.54)',
```

Comment:
confirming the strings around this is intended?

Reply:
Yes, they are. Here, we are using polished's `rgba`.  But, we can use `rgba` provided by css.